src/parseSetupScript.ts

```diff
@@ -0,0 +1,41 @@
+// import ts from 'typescript';
+
+export default async function parseSetupScript(script: HTMLScriptElement, scriptCode: string) {
+    const isTypeScript = (script?.attributes as any)?.lang?.value === 'ts'
+    const sc = document.createElement('script')
+    sc.src = 'https://unpkg.com/typescript/lib/typescript.js'
```

Comment:
can it be imported through the import method, eg: 
```ts
const tsModule = await import('https://unpkg.com/typescript/lib/typescript.js');
const ts = tsModule.ts
```

Reply:
I had tried it, but the typescript lib wasn't compatible